src/api/docgen/view/documentation.ts

```diff
@@ -100,6 +101,25 @@ export class Documentation {
     return documentation;
   }
 
+  public renderToJson(): PackagePageContent {
+    const readme = this.includeReadme
+      ? new Readme(this.transpile, this.assembly, this.submodule)
+          .render()
+          .render()
+      : undefined;
+    const apiReference = new ApiReference(
+      this.transpile,
+      this.assembly,
+      this.submodule
+    );
+
+    return {
+      readme,
+      apiReference: apiReference.renderToJson(),
+      navItems: [],
```

Comment:
Are there plans for this class to generate navItems? Not sure why we would return them here otherwise (unless it is placeholder)

Reply:
placeholder for now, we will derive these from the api reference json + readme me on the client.